gui/public/i18n/ru/translation.ftl

```diff
@@ -0,0 +1,542 @@
+onboarding-automatic_mounting-put_trackers_on-next = Я включил и надел все свои трекеры
+## Tracker manual proportions setup
+onboarding-manual_proportions-back = Вернитесь чтобы сбросить туториал
+onboarding-manual_proportions-title = Ручные пропорции тела
+onboarding-manual_proportions-precision = Регулеровка предсказывания
+onboarding-manual_proportions-auto = Автоматическая калибровка
+
+## Tracker automatic proportions setup
+onboarding-automatic_proportions-back = Вернитесь чтобы сбросить туториал
+onboarding-automatic_proportions-title = Измерьте свое тело
+onboarding-automatic_proportions-description = Чтобы трекеры SlimeVR работали, нам нужно знать длину ваших костей. Эта короткая калибровка измерит его для вас.
+onboarding-automatic_proportions-manual = Ручная калибровка
+onboarding-automatic_proportions-prev_step = Предыдущий шаг
+onboarding-automatic_proportions-put_trackers_on-title = Наденьте ваши трекеры
+onboarding-automatic_proportions-put_trackers_on-description = Чтобы откалибровать ваши пропорции, мы собираемся использовать трекеры, которые вы только что назначили. Включите все свои трекеры, вы можете увидеть, какие из них какие на рисунке справа.
+onboarding-automatic_proportions-put_trackers_on-next = Я надел все свои трекеры
+onboarding-automatic_proportions-preparation-title = Подготовка
+onboarding-automatic_proportions-preparation-description = Поставьте стул прямо позади себя в вашем игровом пространстве. Будьте готовы сесть во время настройки автобона.
+onboarding-automatic_proportions-preparation-next = Я перед стулом
+onboarding-automatic_proportions-start_recording-title = Будьте готовы к движению
+onboarding-automatic_proportions-start_recording-description = Теперь мы собираемся записать некоторые конкретные позы и движения. Они будут запрошены на следующем экране. Будьте готовы начать, когда кнопка будет нажата!
+onboarding-automatic_proportions-start_recording-next = Начать запись
+onboarding-automatic_proportions-recording-title = Запись
+onboarding-automatic_proportions-recording-description-p0 = Запись в процессе...
+onboarding-automatic_proportions-recording-description-p1 = Сделайте эти движенияя:
+onboarding-automatic_proportions-recording-steps-0 = Согните колени несколько раз.
+onboarding-automatic_proportions-recording-steps-1 = Сядьте на стул, затем встаньте.
+onboarding-automatic_proportions-recording-steps-2 = Поверните верхнюю часть тела влево, затем согните вправо.
+onboarding-automatic_proportions-recording-steps-3 = Поверните верхнюю часть туловища вправо, затем согните левую.
+onboarding-automatic_proportions-recording-steps-4 = Покачайтесь, пока таймер не закончится.
+onboarding-automatic_proportions-recording-processing = Обрабатываем результаты...
+# $time (Number) - Seconds left for the automatic calibration recording to finish (max 15)
+onboarding-automatic_proportions-recording-timer = { $time ->
+    [one] секунда осталась
```

Comment:
While the comment above states that the max is 15, it can be changed to a different number.
Therefore I think you should change this to
```
[one] { $time } секунда осталась
```
as I suggested, as otherwise the english translation of:
`21 seconds remaining` will be `секунда осталась`, as would 31, 41 etc.